src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -2020,11 +2020,15 @@ subroutine ocn_compute_KPP_input_fields(statePool, forcingPool, meshPool, timeLe
                               -100.0_RKIND) )
          ! Store the total tracer flux below in nonLocalSurfaceTemperatureFlux for use in the CVMix nonlocal
          ! transport code.  This includes tracer forcing due to thickness
+         ! Note: rainFlux, evaporationFlux, seaIceFreshWaterFlux, icebergFreshWaterFlux 
+         !       all have units of kg m^{-2} s^{-1}, so require division by rho_sw.
+         !       surfaceThicknessFlux, surfaceThicknessFluxRunoff have units of m s^{-1},
+         !       so has no division by rho_sw is needed.
 
          nonLocalSurfaceTracerFlux(indexTempFlux, iCell) = activeTracersSurfaceFlux(indexTempFlux,iCell) &
                  + penetrativeTemperatureFlux(iCell) - penetrativeTemperatureFluxOBL(iCell)  &
                  - fracAbsorbed * (rainFlux(iCell) + evaporationFlux(iCell)) * activeTracers(indexTempFlux,1,iCell)/rho_sw &
-                 - fracAbsorbedRunoff * surfaceThicknessFluxRunoff(iCell)* min(activeTracers(indexTempFlux,1,iCell),0.0_RKIND)/rho_sw &
+                 - fracAbsorbedRunoff * surfaceThicknessFluxRunoff(iCell)* min(activeTracers(indexTempFlux,1,iCell),0.0_RKIND) &
```

Comment:
@vanroekel I added the note above because someone in the future might think `/rho_sw` is missing again. I checked the Registry, and units are all correct there and agree with this correction.